.gitignore

```diff
@@ -1,71 +1,71 @@
-# Visual Studio
-Release
-bin
-obj
-*.ncb
-*.vcproj*
-*.suo
-*.user
-*.sln.cache
-*.manifest
-*.CodeAnalysisLog.xml
-*.lastcodeanalysissucceeded
-_ReSharper.*/
-
-# movies
-*.vqa
-
-# archives
-*.mix
-
-# binaries
-mods/*/*.dll
-mods/*/*.mdb
-/*.dll
-/*.dll.config
-/*.so
-/*.dylib
-/*.pdb
-/*.mdb
-/*.exe
-thirdparty/download/*
-
-# backup files by various editors
-*~
-*.orig
-\#*
-.*.sw?
-
-# Monodevelop
-*.pidb
-*.userprefs
-
-# Mac OS X
-.DS_Store
-
-# XCode
-packaging/osx/launcher/build/
-packaging/osx/launcher/OpenRA.xcodeproj/*.pbxuser
-packaging/osx/launcher/OpenRA.xcodeproj/*.perspectivev3
-packaging/osx/launcher/OpenRA.xcodeproj/*.mode1v3
-temp.c
-temp.o
-temp.s
-OpenRA.Launcher.Mac/build/
-OpenRA.Launcher.Mac/OpenRA.xcodeproj/*.pbxuser
-OpenRA.Launcher.Mac/OpenRA.xcodeproj/*.perspectivev3
-OpenRA.Launcher.Mac/OpenRA.xcodeproj/*.mode1v3
-*.resources
-
-# auto-generated documentation
-DOCUMENTATION.md
-Lua-API.md
-*.html
-
-# StyleCop
-*.Cache
-StyleCopViolations.xml
-
-# SublimeText
-*.sublime-project
-*.sublime-workspace
\ No newline at end of file
+# Visual Studio
+Release
+bin
+obj
+*.ncb
+*.vcproj*
+*.suo
+*.user
+*.sln.cache
+*.manifest
+*.CodeAnalysisLog.xml
+*.lastcodeanalysissucceeded
+_ReSharper.*/
+
+# movies
+*.vqa
+
+# archives
+*.mix
+
+# binaries
+mods/*/*.dll
+mods/*/*.mdb
+/*.dll
+/*.dll.config
+/*.so
+/*.dylib
+/*.pdb
+/*.mdb
+/*.exe
+thirdparty/download/*
+
+# backup files by various editors
+*~
+*.orig
+\#*
+.*.sw?
+
+# Monodevelop
+*.pidb
+*.userprefs
+
+# Mac OS X
+.DS_Store
+
+# XCode
+packaging/osx/launcher/build/
+packaging/osx/launcher/OpenRA.xcodeproj/*.pbxuser
+packaging/osx/launcher/OpenRA.xcodeproj/*.perspectivev3
+packaging/osx/launcher/OpenRA.xcodeproj/*.mode1v3
+temp.c
+temp.o
+temp.s
+OpenRA.Launcher.Mac/build/
+OpenRA.Launcher.Mac/OpenRA.xcodeproj/*.pbxuser
+OpenRA.Launcher.Mac/OpenRA.xcodeproj/*.perspectivev3
+OpenRA.Launcher.Mac/OpenRA.xcodeproj/*.mode1v3
+*.resources
+
+# auto-generated documentation
+DOCUMENTATION.md
+Lua-API.md
+*.html
+
+# StyleCop
+*.Cache
+StyleCopViolations.xml
+
+# SublimeText
+*.sublime-project
+*.sublime-workspace
```

Comment:
do you want it to be CRLF or LF then?
when i do a fresh clone i get CRLF, but the .gitattributes suggests that it should be LF